### ISSUE TITLE
/r works but ! does not

### ISSUE THREAD

**Issue by Zahaqiel** [Rolisteam/DiceParser#151]
At the moment DiceParser seems to be non-responsive to any dice command beginning with ! (ie, **!1d20+3**), but if I use /r and make sure Discord registers a command input (either by selecting the popup above the chat box of "command" or having the "command" box appear automatically in the chat box and filling it in) then it does work.

**/conf show** gives the following:

> [Conf] Server Configuration:
prefix: !
displayMode: markdown
notifyuser: none
answerbot: off

I believe that means the ! commands should be working, but... *shrug*

Kicking and re-adding the bot did not fix the issue.

TL;DR: **/r 1d20+3** works, but **!1d20+3** does not, not sure why.

**Comment by FoolishNoobish** [comment]
This is the exact issue I am having on 2 different servers that I use for D$D.

**Comment by obiwankennedy** [comment]
Due to discord policy we switch to slash commands.
The use of prefix is no longer working.

**Comment by Zahaqiel** [comment]
> 
Oh weird.  A lot of the /help scripts still refer to ! commands.  I'm guessing updating those is a bit of work given the quantity of them.

**Comment by obiwankennedy** [comment]
> How are the alias going to work from this point onward?

Just fine.
